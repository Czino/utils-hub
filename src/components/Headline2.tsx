export const Headline2 = ({ className, children }: React.ComponentProps<'h2'>) => (
  <h2 className={['text-3xl font-bold text-success-2', className].join(' ')}>{children}</h2>
)
