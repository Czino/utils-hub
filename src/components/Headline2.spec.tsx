import { render } from '@testing-library/react'
import { Headline2 } from './Headline2'

describe('Headline2', () => {
  it('should render Headline2 with styles', () => {
    const { asFragment } = render(<Headline2 className="mt-4">Test</Headline2>)
    expect(asFragment()).toMatchSnapshot()
  })
})
