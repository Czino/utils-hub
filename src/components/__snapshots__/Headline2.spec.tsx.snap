// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headline2 should render Headline2 with styles 1`] = `
<DocumentFragment>
  <h2
    class="text-3xl font-bold text-success-2 mt-4"
  >
    Test
  </h2>
</DocumentFragment>
`;
